content/en/docs/concepts/security/pod-security-standards.md

```diff
@@ -475,16 +475,15 @@ Kubelets prior to v1.24 don't enforce the pod OS field, and if a cluster has nod
 {{< /note >}}
 
 ### Restricted Pod Security Standard changes
-Another important change, made in Kubernetes v1.25 is that the  _restricted_ Pod security
+Another important change, made in Kubernetes v1.25 is that the  _Restricted_ policy
```

Comment:
If you want to capitalize the policy name, we should do it everywhere. There are a few other references to the restricted policy, and a bunch to the baseline policy that need capitalization.

Reply:
Capitalized all occurrences of the three policy names